tests/unit/backend/native/models/classification/backbones/test_vision_transformer.py

```diff
@@ -61,3 +61,16 @@ def test_forward(self, config):
         assert len(outs) == 1
         feat = outs[-1]
         assert feat.shape == (1, 197, model.embed_dim)
+
+    def test_peft(self, config):
```

Comment:
It's also beneficial to have separate unit tests for `AttentionWith*` in a standalone file as well, since this test is tied to a particular backbone